server/database/helpers.js

```diff
@@ -42,9 +42,32 @@ const getUserSuperhostStatus = async (userId) => {
   }
 };
 
+//update if ID exists already, insert if not - create and update together
+const updateUserInfo = (id, params) => {
+  const { name, bio, avatarUrl, isSuperhost, identityVerified, languages, responseRate, responseTime } = params;
+  //if any of the params is a falsy value
+  for(let prop in params) {
+    if(!params[prop]) {
+      delete params[prop];
+    }
+  }
```

Comment:
I'm not sure you are using any of the variables declared above when destructuring the passed in `params` object.  Also by definition every `prop` key in the params object will by definition exist.  Let me know if that doesn't make sense.

Reply:
Actually, when I update an existing record, if any of the params is missing, then it overwrites it with null. For example, if I want to update user 1's name from 'John Doe' to 'Jane Doe' but don't want to update any other fields, and i only pass {name: 'Jane Doe'} it would update remaining params with null. But this works lol